src/functions/dialog/index.ts

```diff
@@ -2,8 +2,29 @@
  * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
-import type { Component } from 'vue'
-import { createApp } from 'vue'
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import { createApp, type Component } from 'vue'
```

Comment:
We prefer to always separate type and non-type imports (not included in the old eslint config, using in this repo).

```suggestion
import { type Component } from 'vue'
import { createApp } from 'vue'
```

Reply:
Sure, fixed.